source/_posts/2021-10-06-release-202110.markdown

```diff
@@ -496,8 +496,8 @@ installed, make sure it's part of your manifest.
 
 The following integrations are no longer available as of this release:
 
-- **Trackr** has been removed. The API is no longer available.
-  has therefore been removed. ([@FuzzyMistborn] - [#55917])
+- **Trackr** has been removed. The API is no longer available and
```

Comment:
This is now duplicate, as the title of the list already implies an integration.

Reply:
Ok, but there's still a typo.

Reply:
Oh right!